pkg/cloud_provider/file/file.go

```diff
@@ -1088,6 +1090,7 @@ func cloudInstanceToMultishareInstance(instance *filev1beta1multishare.Instance)
 		MaxCapacityBytes:   instance.MaxCapacityGb * util.Gb,
 		CapacityStepSizeGb: instance.CapacityStepSizeGb,
 		Description:        instance.Description,
+		MaxShareCount:      int(instance.MaxShareCount),
```

Comment:
for this, do we want to add logic to have a default value (10?) if the `instance.MaxShareCount` is not populated?

Reply:
that would again need an if check for feature flag. hence kept that logic in the controller